### DISCUSSION TITLE
I am wondering where "everything I ever copied to clipboard" is saved?

### DISCUSSION THREAD

**Discussion by oujii** [jasonjmcghee/rem#76] [Q&A] [answered]
I can't seem to find it, I've looked in sqlite database and in the folder where "Show me my data" is :)

**Comment by jasonjmcghee** [comment, marked as answer]
Hi @oujii - currently it’s saved as context (if you have the setting enabled- off by default) alongside the ocr extracted text. Currently it is not saved as a separate field. Is that a feature you’d be interested in?

**Reply by oujii** [reply]
Ah I see! Thanks for clarifying! I thought it was a pretty nice thing to save all clipboard stuff, yes I'd be interested! I think this is a great project btw and following it closely :) I wish I could help more with coding, but it's not my strong side.
